tests/phpunit/tests/l10n/getUserLocale.php

```diff
@@ -67,10 +67,6 @@ public function test_site_locale_is_not_affected_on_frontend() {
 	 * @group ms-required
 	 */
 	public function test_user_locale_is_same_across_network() {
-		if ( ! is_multisite() ) {
-			$this->markTestSkipped( 'This test requires Multisite.' );
-		}
```

Comment:
`@group` tags are all nice and dandy, but don't prevent a test from _not_ being run without complying with certain conditions. Only for running them selectively.

This condition should not be removed.